windows/src/test/manual-tests/platform-rules/platformtest.kps

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package>
+  <System>
+    <KeymanDeveloperVersion>10.0.1103.0</KeymanDeveloperVersion>
+    <FileVersion>7.0</FileVersion>
+  </System>
+  <Options>
+    <ExecuteProgram></ExecuteProgram>
+    <MSIFileName></MSIFileName>
+    <MSIOptions></MSIOptions>
+  </Options>
+  <StartMenu>
+    <Folder></Folder>
+    <Items/>
+  </StartMenu>
+  <Info>
+    <Version URL="">1.0</Version>
+    <Name URL="">PlatformTest</Name>
+  </Info>
+  <Files>
+    <File>
+      <Name>PlatformTest.kmx</Name>
+      <Description>Keyboard PlatformTest</Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.kmx</FileType>
+    </File>
+    <File>
+      <Name>platformtest.js</Name>
+      <Description>File platformtest.js</Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.js</FileType>
+    </File>
+  </Files>
+  <Keyboards>
+    <Keyboard>
+      <Name>PlatformTest</Name>
+      <ID>platformtest</ID>
+      <Version>1.0</Version>
+      <Languages/>
```

Comment:
Adding a reminder to include a language tag so we can test the .kmp on the mobile apps.